demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -514,7 +514,14 @@ def _collect_playbooks_dependencies(pack_playbooks: list, id_set: dict, verbose:
                 playbook_data.get('implementing_playbooks', []),
                 skippable_tasks,
                 id_set['playbooks'],
-                exclude_ignored_dependencies
+                exclude_ignored_dependencies,
+            ))
+
+            playbook_dependencies.update(PackDependencies._differentiate_playbook_implementing_objects(
```

Comment:
Same as said in ID set, need to consider dependencies for integrations/scripts calling getList/setList (if They're found in the ID set of course).

Probably only getList is important, setList shouldn't have dependency I guess